keyboards/crkbd/keymaps/thunderbird2086/rgb.c

```diff
@@ -0,0 +1,58 @@
+#if defined(RGBLIGHT_ENABLE) || defined(RGB_MATRIX_ENABLE)
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
done